lisa/operating_system.py

```diff
@@ -396,19 +396,21 @@ def capture_system_information(self, saved_path: Path) -> None:
                 except Exception as identifier:
                     self._node.log.debug(f"error on get boot time: {identifier}")
 
-            from lisa.tools import Chmod, Find
+            file_list = []
+            if self._node.capture_azure_information:
+                from lisa.tools import Chmod, Find
 
-            find_tool = self._node.tools[Find]
-            file_list = find_tool.find_files(
-                self._node.get_pure_path("/var/log/azure/"),
-                file_type="f",
-                sudo=True,
-                ignore_not_exist=True,
-            )
-            if len(file_list) > 0:
-                self._node.tools[Chmod].update_folder(
-                    "/var/log/azure/", "a+rwX", sudo=True
+                find_tool = self._node.tools[Find]
+                file_list = find_tool.find_files(
+                    self._node.get_pure_path("/var/log/azure/"),
+                    file_type="f",
+                    sudo=True,
+                    ignore_not_exist=True,
                 )
+                if len(file_list) > 0:
+                    self._node.tools[Chmod].update_folder(
+                        "/var/log/azure/", "a+rwX", sudo=True
+                    )
             file_list.append("/etc/os-release")
```

Comment:
/etc/os-release can be collected, since it is not azure specific platform

Reply:
Yes, it is not in the capture_azure_information condition branch.